src/proteus/atmos_clim/agni.py

```diff
@@ -360,8 +364,8 @@ def run_agni(atmos, loops_total:int, dirs:dict, OPTIONS:dict, hf_row:dict):
 
         # Try solving temperature profile
         agni_success = jl.AGNI.solver.solve_energy_b(atmos,
-                            sol_type=OPTIONS["atmosphere_surf_state"],
-                            chem_type=OPTIONS["atmosphere_chemistry"],
+                            sol_type=config.atmos_clim.surf_state,
```

Comment:
AGNI still requires `sol_type` to be passed as an integer.
The mapping is: `('mixed_layer', 'fixed', 'skin')` -> `(0,1,2)`.

Reply:
Fixed using a new property on config.atmos_clim